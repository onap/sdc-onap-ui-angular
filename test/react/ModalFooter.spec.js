import React from 'react';
import ModalFooter from '../../src/react/ModalFooter.js';

import renderer from 'react-test-renderer';

describe('ModalFooter', () => {
	test('basic test', () => {
		const footer = renderer.create(<ModalFooter closeButtonText='Ok'/>).toJSON();        
		expect(footer).toMatchSnapshot();
	});
});