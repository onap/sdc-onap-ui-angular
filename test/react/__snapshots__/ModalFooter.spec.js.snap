// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalFooter basic test 1`] = `
<div
  className="sdc-modal__footer"
>
  <div>
    <button
      className="sdc-button sdc-button__primary   "
      disabled={false}
      onClick={undefined}
    >
      Ok
    </button>
  </div>
</div>
`;
